src/ui/views/Dashboard/components/Feedback/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { DrawerProps } from 'antd';
+import clsx from 'clsx';
+import React from 'react';
+import { browser } from 'webextension-polyfill-ts';
+
+import IconArrowRight from 'ui/assets/arrow-right-gray.svg';
+import { Field, Popup } from 'ui/component';
+import './style.less';
+import IconDirectMessage from 'ui/assets/feedback-popup/entry-hi.svg';
+import IconProposal from 'ui/assets/feedback-popup/entry-proposal.svg';
+import IconItemLink from 'ui/assets/feedback-popup/item-link.svg';
+
+interface SettingsProps {
+  visible?: boolean;
+  onClose?: DrawerProps['onClose'];
+}
+
+const FeedbackPopup = ({ visible, onClose }: SettingsProps) => {
+  const renderData = [
+    {
+      leftIcon: IconDirectMessage,
+      content: 'Direct Message',
+      description: 'Chat with Rabby Wallet Official on DeBank',
+      rightIcon: <img src={IconItemLink} className="icon icon-item-link" />,
+      onClick: () => {
+        browser.tabs.create({ url: 'https://debank.com/hi/0a110032' });
```

Comment:
https://github.com/RabbyHub/Rabby/pull/1498/files#diff-2c73e787e6a4aa44ae3244eba3db66ff7558536ffdc61b7b576aa512aca3bd28R26
You can use `openInTab` util for this